lib/honeydew.ex

```diff
@@ -363,12 +363,17 @@ defmodule Honeydew do
     supervisor_opts =
       opts
       |> Keyword.get(:supervisor_opts, [])
-      |> Keyword.put_new(:id, {:queue, name})
+      |> Keyword.put_new(:restart, :permanent)
```

Comment:
How come you added these? Aren't `:permanent` and `:infinity` the defaults for these options?

Reply:
Hmm yes. Perhaps I was being too explicit?

Reply:
I think it might be best to remove them, including them indicates to me that they were set for a reason.

Reply:
👍  Done